### ISSUE TITLE
Manual Finetuning and hugging_face finetuning don't have the same behaviour

### ISSUE THREAD

**Issue by LaurineMir** [swerik-project/pretraining_plm#2]
I have implemented two different methods of finetuning one with the trainer from hugging face (finetining_hugging_face) and one manually(finetuning_manual) (it enables to have a deterministic random masking). The problem is that the two methods don't give the same results at all :

- in the manual one . the eval loss increases very quickly
- in the trainer : the eval loss decreases then stagnates
